src/main.rs

```diff
@@ -45,6 +45,13 @@ enum LedDisableTarget {
     All,
 }
 
+#[derive(Debug, Copy, Clone)]
+/// Keep track which sensors should be measured
+struct MeasurementPlan {
+    measure_shtc3: bool,
```

Comment:
In other locations (e.g. `Resources`) we only used `sht` as identifier.

```suggestion
    measure_sht: bool,
```

(Needs updates in other locations as well, so it's probably easier to do the changes by hand instead of applying the suggestion.)